sound/soc/sof/intel/hda-dai.c

```diff
@@ -0,0 +1,136 @@
+// SPDX-License-Identifier: (GPL-2.0 OR BSD-3-Clause)
+/*
+ * This file is provided under a dual BSD/GPLv2 license.  When using or
+ * redistributing this file, you may do so under either license.
+ *
+ * Copyright(c) 2018 Intel Corporation. All rights reserved.
+ *
+ * Authors: Keyon Jie <yang.jie@linux.intel.com>
+ */
+
+#include <sound/pcm_params.h>
+#include "../sof-priv.h"
+
+/* common dai driver for skl+ platforms */
+static const struct snd_soc_dai_ops skl_dai_ops = {
```

Comment:
why is this needed? Can we add later when it's actually doing something?

Reply:
are you meaning this hda-dai.c file? this is suggested by Liam to add, and it addressing your comment about common dai driver table for SKL/APL/CNL also.